IIIT Hyderabad Faculty Update

narenakash opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (up to three months, as processing has moved to a quarterly cadence).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv")._

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. _Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org)._

1. _If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below)._

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [X] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

narenakash committed:
Makarand Tapaswi joined IIIT Hyderabad

narenakash committed:
Added Deva Priyakumar to IIIT Hyderabad

narenakash committed:
Ponnurangam Kumaraguru joined IIIT Hyderabad

Updated the institution and home page. 
Removed redundant entry.

narenakash committed:
Added Ravi Kiran Sarvadevabhatla to IIIT Hyderabad

narenakash committed:
Charu Sharma joined IIIT Hyderabad

narenakash committed:
Added Spandan Roy to IIIT Hyderabad

narenakash committed:
Merge pull request #1 from narenakash/patch-9

Charu Sharma joined IIIT Hyderabad

narenakash committed:
Merge pull request #6 from narenakash/patch-5

Added Ravi Kiran Sarvadevabhatla to IIIT Hyderabad

narenakash committed:
Merge pull request #2 from narenakash/patch-1

Makarand Tapaswi joined IIIT Hyderabad

narenakash committed:
Merge pull request #5 from narenakash/patch-4

Ponnurangam Kumaraguru joined IIIT Hyderabad

narenakash committed:
Merge pull request #3 from narenakash/patch-11

Added Spandan Roy to IIIT Hyderabad

narenakash committed:
Merge pull request #4 from narenakash/patch-3

Added Deva Priyakumar to IIIT Hyderabad

emeryberger commented on csrankings-p.csv:
```diff
@@ -1001,8 +1001,7 @@ Polyvios Pratikakis,University of Crete,https://www.csd.uoc.gr/~polyvios,ACXEa94
 Pong P. Chu,Cleveland State University,https://academic.csuohio.edu/chu_p,NOSCHOLARPAGE
-Ponnurangam Kumaraguru,IIIT Delhi,https://www.iiitd.ac.in/pk,MfzQyP8AAAAJ
-Ponnurangam Kumarguru,IIIT Delhi,https://www.iiitd.ac.in/pk,MfzQyP8AAAAJ
+Ponnurangam Kumaraguru,IIIT Hyderabad,https://www.iiit.ac.in/people/faculty/PKguru/,MfzQyP8AAAAJ
```
Both aliases for this author *must* be included, per the instructions.

emeryberger commented on csrankings-u.csv:
```diff
@@ -1,5 +1,6 @@
 name,affiliation,homepage,scholarid
 U Kang,Seoul National University,https://datalab.snu.ac.kr/~ukang,NVTb6RYAAAAJ
+U. Deva Priyakumar,IIIT Hyderabad,https://www.iiit.ac.in/people/faculty/deva/,dnaEkO0AAAAJ
```
Not in CS. Please clarify how they meet the criteria.

emeryberger replied:
This still needs to be addressed.

narenakash committed:
Ponnurangam Kumarguru joined IIIT Hyderabad

Added the required alias of Ponnurangam Kumaraguru

narenakash committed:
Removed U. Deva Priyakumar from PR

I will create a separate issue for the inclusion of U. Deva Priyakumar from CCNSB.
